1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -18,6 +20,20 @@ const BIRTHDAYS = [
 
 function findFirstJulyBDay(birthdays) {
     // TODO
+     let i=0;
+     while(i<birthdays.length){
+    //     //using includes()methods
```

Comment:
Please remove the dead code(code is not being used). You are not applying code beatification(prettier or liniting), clean code improves the readability

Reply:
How would you fix the problem next time?
For now please refer following guide regarding linting,

https://syllabus.codeyourfuture.io/guides/code-style-guide

Reply:
> Please remove the dead code(code is not being used). You are not applying code beatification(prettier or liniting), clean code improves the readability

Thanks for your good comments and I'll follow your comments in the future.
Because it's an exercise, I tried different solutions and commented optional solutions in the codes.